app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -79,8 +79,7 @@
   <%= validated_form_for :doc_auth,
                          url: url_for,
                          method: 'put',
-                         html: { autocomplete: 'off', class: 'margin-top-2 js-consent-continue-form' } do |f| %>
-    <br/>
+                         html: { autocomplete: 'off', class: 'margin-top-4 padding-bottom-1 js-consent-continue-form' } do |f| %>
```

Comment:
If the idea is there should be 5 units of margin around the button, maybe we can do...

```suggestion
                         html: { autocomplete: 'off', class: 'margin-y-5 js-consent-continue-form' } do |f| %>
```

This seems to collapse as expected.

The only problem is that there's extra padding on the last list item on the welcome step, but it seems like we can / should remove that.